Gemfile.lock

```diff
@@ -60,7 +60,6 @@ GEM
       scrypt (>= 1.2, < 4.0)
     authlogic-oid (1.0.4)
```

Comment:
Please remove Gemfile.lock from your PR, as we update it only when required. Thanks!